packages/client/src/generation/generateClient.ts

```diff
@@ -272,7 +272,6 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
 
   // if users use a custom output dir
   if (copyRuntime || !path.resolve(outputDir).endsWith(`@prisma${path.sep}client`)) {
-    // TODO: Windows, / is not working here...
```

Comment:
Outdated, refers the previous line. Introduced in https://github.com/prisma/prisma/commit/871cc4d13856f2697d8b8772c569f57fb2d3d8ec, fixed in https://github.com/prisma/prisma/commit/e9647573c5e314fc6cdb3ebe1d839bfc5a3cfcf2.